.github/scripts/runner_determinator.py

```diff
@@ -429,7 +429,7 @@ def is_user_opted_out(user: str, user_optins: UserOptins, experiment_name: str)
 
 def get_runner_prefix(
     rollout_state: str,
-    workflow_requestors: Iterable[str],
+    workflow_requesters: Iterable[str],
```

Comment:
I don't think there a strong rule here to be frank, see https://www.grammarpalette.com/requester-or-requestor-which-spelling/